conf/battle/client.conf

```diff
@@ -169,3 +169,10 @@ macro_detection_punishment: 0
 // Amount of time in minutes that the punishment type is active for. Use 0 for infinite.
 // Official: 0
 macro_detection_punishment_time: 0
+
+// Should the damage timing be synchronized between the client and server?
+// This is not official behavior, but it removes the position lag after being hit.
+// This setting only affects normal monster attacks, unless you set "delay_battle_damage" in battle.conf to "no".
+// If you do, most skills will also be in-sync as skills usually show their damage immediately.
+// Tired of Dark Illusion hitting you 5 seconds too late? Then turn this on. (Note 1)
+synchronize_damage: no
```

Comment:
Should be moved to conf/battle/battle.conf right after delay_battle_damage

Reply:
Originally placed it here because I thought while delay_battle_damage affects more the functionality, the new feature more affects the client display. But considering that both can affect the amotion value and also have some synergy, it makes sense, I'll move it together.